package.json

```diff
@@ -50,7 +50,7 @@
   },
   "scripts": {
     "lint": "standard --fix commands/* helpers/* libs/* index.js",
-    "test": "jest",
+    "test": "env LOG_LEVEL=info jest",
```

Comment:
Debug code I suppose

Reply:
No, I made the change otherwise the test have a lot of logs